### ISSUE TITLE
tails server environment variables formatting reacts badly with some versions of docker-compose

### ISSUE THREAD

**Issue by voutasaurus** [kiva/protocol-integration-tests#25]
Background:
An error from one of our pipelines consuming this repo:
```
invalid interpolation format for services.tails-server.environment.[]: "$(docker run --rm --net=host eclipse/che-ip)". You may need to escape any $ with another $.

Exited with code exit status 15

```

Action required: Assess the feasibility of removing or simplifying these environment variables in the tails server section of the docker compose file in this repo:
```
environment:
      - MSYS_NO_PATHCONV=1
      - DOCKERHOST=${APPLICATION_URL-$(docker run --rm --net=host eclipse/che-ip)}
      - GENESIS_URL=${GENESIS_URL:-http://$DOCKERHOST:9000/genesis}
      - STORAGE_PATH=${STORAGE_PATH:-/tmp/tails-files}
      - LOG_LEVEL=${LOG_LEVEL:-INFO}
      - TAILS_SERVER_URL=${TAILS_SERVER_URL:-http://$DOCKERHOST:6543}
```

Bonus: if you want to switch to the BC provided tails server image https://github.com/bcgov/indy-tails-server/pkgs/container/tails-server

**Comment by ghost** [comment]
Taking the easiest path forward to unblock the pipeline. This seems to be a bug in the way docker-compose works on linux, so the easiest thing to do is keep using the same environment variables we're using and simply move them to a `.env` file. I'll create a separate investigation issue for looking into how to minimize these env vars and what we would gain from switching to the BC-provided tails server image.

**Comment by ghost** [comment]
Created a new issue to explore using the BCGov Tails Server image and look into a general environment variable clean up: https://github.com/kiva/protocol-integration-tests/issues/27
